.travis.yml

```diff
@@ -84,7 +84,7 @@ script:
       ./node_modules/.bin/grunt flow
 
       ALL_FILES=`find src -name '*.js' | grep -v umd/ | grep -v __tests__ | grep -v __mocks__`
-      COUNT_ALL_FILES=`echo $ALL_FILES | wc -l`
```

Comment:
Omg, thanks! I couldn't figure out what I did wrong. Of course, sending it to the tokenizer of bash like this is going to interpret them as different arguments and be in one line when echo sees it! Good catch!